web/src/test/java/com/epam/rft/atsy/cucumber/application/NewApplicationPopupStepDefs.java

```diff
@@ -0,0 +1,217 @@
+    dropdown.selectByVisibleText(DIRECT_IN_HUN);
+  }
+
+  @And("^leaves the comment blank$")
+  public void leaves_the_comment_blank() throws Throwable {
+    // Do nothing
+  }
+
+  @When("^the user clicks on the Jelentkezés mentése button$")
+  public void the_user_clicks_on_the_save_new_apply_button() throws Throwable {
+    webDriver.findElement(By.id(SAVE_NEW_APPLY_BUTTON)).click();
+  }
+
+  @Then("^New application popup disappears$")
+  public void new_application_popup_disappears() throws Throwable {
+    try {
+      webDriver.findElement(By.id(POP_UP_CONTENT));
+      // Wrong behaviour
+      Assert.fail();
+    } catch (NoSuchElementException e) {
+      // Expect behaviour
+    }
+  }
+
+  @And("^the Application page is displayed$")
+  public void the_application_page_is_displayed() throws Throwable {
+    assertThat(webDriver.getCurrentUrl(), equalTo("http://localhost:8080/atsy/secure/candidate/1"));
+  }
+
+  @And("^application is listed with all details$")
+  public void application_is_listed_with_details() throws Throwable {
+    Date presentDate = currentDatePlus(5);
+    WebElement table = webDriver.findElement(By.id(APPLICATIONS_TABLE));
+    List<WebElement> lastRowInTheTable = getValuesFromTheLastRowFromTheTable(table);
+
+    String positionName = lastRowInTheTable.get(COLUMN_ZEROTH).getText();
+    Date addedDate = SIMPLE_DATE_FORMAT.parse(lastRowInTheTable.get(COLUMN_FIRST).getText());
+    Date lastModifiedDate = SIMPLE_DATE_FORMAT.parse(lastRowInTheTable.get(COLUMN_SECOND).getText());
+    String stateName = lastRowInTheTable.get(COLUMN_THIRD).getText();
+
+    assertThat(positionName, equalTo(DEVELOPER_IN_HUN));
+    assertThat(addedDate, lessThanOrEqualTo(presentDate));
+    assertThat(lastModifiedDate, lessThanOrEqualTo(presentDate));
+    assertThat(stateName, equalTo(NEW_APPLY_IN_HUN));
+  }
+
+
+  @And("^the user enters comment Megjegyzés$")
+  public void the_user_enters_comment_Remark() throws Throwable {
+    webDriver.findElement(By.id(DESCRIPTION)).sendKeys(COMMENT_IN_HUN);
+  }
+
+  @And("^the user doesn't select a position$")
+  public void the_user_does_not_select_a_position() throws Throwable {
+    // Do nothing
+  }
+
+  @Then("^the Kérem válasszon egy pozíciót message appears under the position selector$")
+  public void the_please_choose_one_position_message_appears_under_the_position_selector() throws Throwable {
+    String positionDropDownErrorMessage = webDriver.findElement(By.id("position_error")).getText();
+    assertThat(positionDropDownErrorMessage, equalTo(POSITION_DROP_DOWN_ERROR_MESSAGE_IN_HUN));
+  }
+
+  @And("^the user doesn't select a source$")
+  public void the_user_does_not_select_a_source() throws Throwable {
+    // Do nothing
+  }
+
+  @Then("^the Kérem válasszon jelentkezési forrást message appears under the source selector$")
+  public void the_please_choose_an_application_source_message_appears_under_the_source_selector() throws Throwable {
```

Comment:
In case the error message changes, the feature file and the step definitions are needed to be changed.

To avoid double work, create a parameter for this step which extracts the 'Kérem válasszon jelentkezési forrást' string from the step name, and use that value in the script.